packages/web-components/fast-components/src/tree-item/tree-item.styles.ts

```diff
@@ -95,232 +96,174 @@ export const treeItemStyles: (
 ) => ElementStyles = (context: ElementDefinitionContext, definition: TreeItemOptions) =>
     css`
     ${display("block")} :host {
-        contain: content;
-        position: relative;
-        outline: none;
-        color: ${neutralForegroundRest};
-        background: ${neutralFillStealthRest};
-        cursor: pointer;
-        font-family: ${bodyFont};
-        --expand-collapse-button-size: calc(${heightNumber} * 1px);
-        --tree-item-nested-width: 0;
-    }
-
-    :host(:focus) > .positioning-region {
-        outline: none;
-    }
-
+      contain: content;
+      position: relative;
+      outline: none;
+      color: ${neutralForegroundRest};
+      fill: currentcolor;
+      cursor: pointer;
+      font-family: ${bodyFont};
+      --expand-collapse-button-size: calc(${heightNumber} * 1px);
+      --tree-item-nested-width: 0;
```

Comment:
custom properties should be sorted at the top of the property list

Reply:
updated